### DISCUSSION TITLE
Disable gameobject

### DISCUSSION THREAD

**Discussion by mikitakuzmichkin** [gilzoide/unity-flex-ui#9] [General]
When the gameobject is disabled, the flexlayout does not change

**Comment by mikitakuzmichkin** [comment]
don't work only in editor mode

**Comment by gilzoide** [comment]
Say @mikitakuzmichkin , why would you want to update the layout of a disabled object? What's your use case for this?

**Comment by gilzoide** [comment]
Oh, I guess you were talking about the same bug as #18, sorry if I misunderstood it.
It is fixed now, so I'll be closing this discussion. Feel free to reopen if necessary.
